source/reference/minio-mc/mc-support-top-net.rst

```diff
@@ -0,0 +1,71 @@
+=======================
+``mc support top net``
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc support top net
+
+Syntax
+------
+
+.. start-mc-support-top-net-desc
```

Comment:
Stretch scope here - we have an admonition on `mc support` about these requiring SUBNET, but we don't have that on subpages.

We should maybe have it on all the subpages right after the description.

Reply:
I added it, though I need to double check because i think at least `proxy` does NOT require the registration.